mrp_bom_product_detail/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Bom product details
+===================
+
+This module adds information product price and product stock on bill of material tree view and bom component.
```

Comment:
In general try to keep the sentences around no more 80 characters (nice to have)